<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Indexor</class>
 <widget class="QWidget" name="Indexor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Indexor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Choose CSV index file:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="csv_combobox"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Choose data root:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="path_label">
     <property name="text">
      <string>Selected path........</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="select_folder_button">
     <property name="text">
      <string>Select Folder</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="start_button">
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progress">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
